dev/release/test-helper.rb

```diff
@@ -78,13 +78,16 @@ def detect_versions
     @snapshot_version = cpp_cmake_lists.read[/ARROW_VERSION "(.+?)"/, 1]
     @snapshot_major_version = @snapshot_version.split(".")[0]
     @release_version = @snapshot_version.gsub(/-SNAPSHOT\z/, "")
+    @release_minor_version = @release_version.split(".")[0]+"."+@release_version.split(".")[1]
```

Comment:
This is not a minor version. This is a "major + minor" version.
How about (backward) "compatible" version?

```ruby
@release_compatible_version = @release_version.split(".")[0, 2]).join(".")
```